impl/src/main/java/org/jboss/weld/logging/ValidatorLogger.java

```diff
@@ -273,8 +273,9 @@ public void log(Object... params) {
     @Message(id = 1468, value = "Method {0} defined on class {1} is not defined according to the specification. It is annotated with @{2} but it does not have a {3} return type.\n\tat {4}\n  StackTrace", format = Format.MESSAGE_FORMAT)
     DefinitionException interceptorMethodDoesNotHaveVoidReturnType(Object param1, Object param2, Object param3, Object param4, Object stackElement);
 
-    @Message(id = 1469, value = "Method {0} defined on class {1} is not defined according to the specification. It is annotated with @{2} but it does not have zero parameters.\n\tat {3}\n  StackTrace", format = Format.MESSAGE_FORMAT)
-    DefinitionException interceptorMethodDoesNotHaveZeroParameters(Object param1, Object param2, Object param3, Object stackElement);
+    @LogMessage(level = Level.WARN)
+    @Message(id = 1469, value = "Method {0} defined on class {1} is not defined according to the specification. It is annotated with @{2} but it does not have zero parameters.\n", format = Format.MESSAGE_FORMAT)
```

Comment:
There is no stackTrace. Do you think it's worth of telling the line number of the corresponding method? I thought the name of the method is sufficient.